postgres/datadog_checks/postgres/postgres.py

```diff
@@ -80,10 +80,11 @@ def _get_debug_tags(self):
 
     def _get_service_check_tags(self):
         host = self.resolved_hostname if self.resolved_hostname is not None else self._config.host
-        service_check_tags = ["host:%s" % host]
-        service_check_tags.extend(self._config.tags)
-        service_check_tags = list(set(service_check_tags))
-        return service_check_tags
+        service_check_tags = {"postgres_host:%s" % host}
+        if not self.disable_generic_tags:
+            service_check_tags.add("host:%s" % host)
```

Comment:
I agree for `server` but `host` is not a generic tag. It has a specific meaning that should refer to the `host` of the database.